ts/components/UserDetailsDialog.tsx

```diff
@@ -30,6 +31,9 @@ export class UserDetailsDialog extends React.Component<Props> {
 
   public render() {
     const i18n = this.props.i18n;
+    const isRss =
+      window.ConversationController.get(this.props.pubkey).attributes.type ===
+      'group';
```

Comment:
Would be cleaner to pass `isRss` using the props. it's in the file `background.js`, line 1060~ in function `Whisper.events.on('onShowUserDetails'`